PF_inversion_20190907.py

```diff
@@ -363,6 +424,18 @@ def createLocalMesh(rxLoc, ind_t):
 else:
     mesh = meshInput
 
+if show_graphics:
```

Comment:
That's great, we can add more arguments to those variables later to control the plot.